cmd/trivy.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"dynamic-buildkite-template/generator"
+)
+
+var (
+	trivyPluginConfig generator.TrivyPluginConfig
+	flagPrefix        = "trivy-"
+)
+
+func init() {
+	generateCmd.Flags().Int(flagPrefix+"exit-code", 0, "Controls whether the security scan is blocking or not for trivy buildkite plugin")
+	generateCmd.Flags().String(flagPrefix+"timeout", "", "Controls the maximum amount of time a scan will run for trivy buildkite plugin")
```

Comment:
We should set a reasonable default

Reply:
completed , currently passed in minutes, this has to be change exact timeout in my future development.